### PR TITLE
Feature/24131 simple conf

### PR DESCRIPTION
**Pull request by robertjensen**
https://redmine.magenta-aps.dk/issues/24131

- [x] Have you updated the documentation?
- [x] Have you performed a smoke test of the application?
- [ ] Have you written tests for your changes?
- [x] Have you updated the release notes?

Did not write new tests, existing tests cover the fact the the module does not break functionality and new functionality is only visible in the UI, and thus hard to test. Existing tests are updated to accept the new syntax, though.

Documentation is currently in a separate file, awaiting merge of Christoffer's branch with updated docs.

### CODE REVIEW

File: backend/mora/settings.py [hunk 1]
```diff
@@ -44,4 +44,13 @@
 
 PROD_MODE = False
 
+try:
+    with open('backend/mora/user_settings.json') as f:
```
Comment (cmoesgaard) [inline backend/mora/settings.py]: The path for `user_settings` should optionally be configurable, e.g. with an environment variable.
This is in part a wish from the ops people, so the config file can exist separately from the code, which makes deployment and maintenance slightly easier.

(Ideally, we should do this for the location of the main settings file as well, but I think that's out of scope for this branch)

File: backend/mora/user_settings.json [hunk 2]
```diff
@@ -1,4 +1,12 @@
-{'orgunit':
-   {'show_family_line': True,
-    'show_bvn': True}
+{
```
Comment (cmoesgaard) [inline backend/mora/user_settings.json]: This file is described in the docs, and we have some sensible defaults, so I don't think we should include it.

File: configuration.rst [hunk 3]
```diff
@@ -0,0 +1,42 @@
+Configuration module
```
Comment (cmoesgaard) [inline configuration.rst]: There are a few typos
indcates, appply, examplel

File: docs/dev/configuration.rst [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+It is possible to perform simple configuration of the MO frontend using the
+configuation file found in ``backend/mora/user_settings.json`` or in the file
+indicated in the env-variable USER_SETTINGS.
+
+This file could look something like this:
+
+    .. sourcecode:: json
+
+    {
+	"orgunit": {
```
Comment (danchr) [inline docs/dev/configuration.rst]: That unindent completely breaks rendering — you can inspect the generated output using `make -C docs` with your venv activated.

File: frontend/src/organisation/OrganisationDetail.vue [hunk 5]
```diff
@@ -2,12 +2,11 @@
   <div class="card">
     <div class="card-body">
       <h4 class="card-title">
-        <icon name="users" /> {{orgUnit.name}}
+        <icon name="users" /> {{orgUnitInfo.name}}
```
Comment (danchr) [inline frontend/src/organisation/OrganisationDetail.vue]: This renaming seems a bit arbitrary — from the perspective of the frontend, that object is indeed the unit.

File: backend/tests/test_integration_org_unit.py [hunk 6]
```diff
@@ -33,6 +33,9 @@ def test_org_unit_temporality(self):
                     'location': ('Overordnet Enhed/Humanistisk fakultet/' +
                                  'Historisk Institut'),
                     "user_key": "frem",
+                    'user_settings': {'orgunit': {'show_bvn': False,
+                                                  'show_location': True,
+                                                  'show_roles': True}},
```
Comment (danchr) [inline backend/tests/test_integration_org_unit.py]: I'm not fond of including this with each and every request — it seems a bit too specific to the UI. Considering that we don't expect the configuration to change, we should consider just exposing it, and let the frontend determine whether to show anything. As it happens, you recently made the change to give it the entire tree, right?

File: docs/dev/configuration.rst [hunk 7]
```diff
@@ -0,0 +1,43 @@
+Configuration module
+=====================
+
+Front-end configuration
+------------
+
+It is possible to perform simple configuration of the MO frontend using the
+configuation file found in ``backend/mora/user_settings.json`` or in the file
```
Comment (danchr) [inline docs/dev/configuration.rst]: If we retain this, we should add, and mention, a `user_settings-example.json` just as we have `mora-example.json`.

File: frontend/src/organisation/OrganisationDetailTabs.vue [hunk 8]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div>
+  <div v-if="settings">
```
Comment (danchr) [inline frontend/src/organisation/OrganisationDetailTabs.vue]: Eh, why should we hide all tabs if we have no settings? That seems rather unusual…

**Pull request merged**